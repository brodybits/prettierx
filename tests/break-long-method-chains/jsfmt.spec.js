run_spec(__dirname, ["babel", "babel-flow", "typescript"]);
