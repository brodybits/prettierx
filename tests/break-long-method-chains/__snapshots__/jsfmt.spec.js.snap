// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`long-method-chain-test1.js 1`] = `
====================================options=====================================
parsers: ["babel", "babel-flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
obj.first(a).second(b).third(c)

=====================================output=====================================
obj.first(a).second(b).third(c);

================================================================================
`;
